<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DADialogDataFrameSeriesSelector</class>
 <widget class="QDialog" name="DADialogDataFrameSeriesSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DA::DADataManagerComboBox" name="comboBoxDatas"/>
       </item>
       <item>
        <widget class="DA::DAPyDataframeColumnsListWidget" name="listWidgetColumns"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Data preview</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DA::DAPySeriesTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DA::DADataManagerComboBox</class>
   <extends>QComboBox</extends>
   <header>DADataManagerComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>DA::DAPyDataframeColumnsListWidget</class>
   <extends>QListWidget</extends>
   <header>DAPyDataframeColumnsListWidget.h</header>
  </customwidget>
  <customwidget>
   <class>DA::DAPySeriesTableView</class>
   <extends>QTableView</extends>
   <header>DAPySeriesTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DADialogDataFrameSeriesSelector</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>900</x>
     <y>754</y>
    </hint>
    <hint type="destinationlabel">
     <x>19</x>
     <y>761</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DADialogDataFrameSeriesSelector</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>959</x>
     <y>747</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>714</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
